app/_gateway_entities/plugin.md

```diff
@@ -61,6 +55,75 @@ To start creating your own plugins, review the Getting Started documentation, or
 * Plugin Development Kit reference
 * Other Language Support
 
+## Scoping plugins
+
+You can run plugins in various contexts, depending on your environment needs.
+Each plugin can run globally, or be scoped to some combination of the following:
+* [Gateway Services](/gateway/entities/service/)
+* [Routes](/gateway/entities/route/)
+* [Consumers](/gateway/entities/consumer/)
+* [Consumer Groups](/gateway/entities/consumer-group/)
+
+Using scopes, you can customize how {{site.base_gateway}} handles functions in your environment, 
+either before a request is sent to your backend services or after it receives a response.
+For example, if you apply a plugin to a single [**Route**](/gateway/entities/route/), 
+that plugin will only trigger when a request matches the Route's specific path.
+
+### Global scope
+
+A global plugin is not associated to any Service, Route, Consumer, or Consumer Group is 
+considered global, and will be run on every request, regardless of any other configuration.
+
+* In self-managed {{site.ee_product_name}}, the plugin applies to every entity in a given Workspace.
+* In self-managed {{site.ce_product_name}}, the plugin applies to your entire environment.
```

Comment:
{{site.ce_product_name}} needs to be added as a variable. Currently, it's blank on the rendered preview.